gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/LocalGcdHelper.java

```diff
@@ -62,17 +64,85 @@
   public static final String DEFAULT_PROJECT_ID = "projectid1";
   public static final int PORT = 8080;
   private static final String GCD = "gcd-v1beta2-rev1-2.1.2b";
```

Comment:
Agreed and understood. My question is: "Where else do we currently use the variable `GCD_VERSION` that means the same thing conceptually?"

For example, the `"v1beta2"` string we use for API calls doesn't really mean the same thing as the `GCD_VERSION` variable here, right? I wouldn't expect LocalGcdHelper's variables to be used in the Datastore implementation...